tasklist/client/src/common/i18n/locales/de.json

```
@@ -33,6 +33,9 @@
     "loginPasswordFieldPlaceholder": "Passwort",
     "loginLoggingInMessage": "Anmelden",
     "loginButtonLabel": "Anmelden",
+    "forbiddenPageTitle": "Sie haben keinen Zugriff auf diese Komponente",
+    "forbiddenPageDesc": "Es scheint, dass Sie nicht über die erforderlichen Berechtigungen verfügen, um auf diese Komponente zuzugreifen. <strong>Bitte kontaktieren Sie Ihren Cluster-Administrator</strong>, um Zugriff anzufordern.",
+    "forbiddenPageLinkLabel": "Erfahren Sie mehr über Rollen und Berechtigungen",
```

@pedesen are these translations good enough?

---

"dass" sounds a bit weird to me in this case. This is a bit smoother:

```suggestion
    "forbiddenPageTitle": "Sie haben keinen Zugriff auf diese Komponente",
    "forbiddenPageDesc": "Es scheint, als hätten Sie nicht die erforderlichen Berechtigungen, um auf diese Komponente zuzugreifen. <strong>Bitte kontaktieren Sie Ihren Cluster-Administrator</strong>, um Zugriff anzufordern.",
    "forbiddenPageLinkLabel": "Erfahren Sie mehr über Rollen und Berechtigungen",
```

---

Alternative, more formal:

```suggestion
    "forbiddenPageTitle": "Sie haben keinen Zugriff auf diese Komponente",
    "forbiddenPageDesc": "Es scheint, als ob Sie nicht über die erforderlichen Berechtigungen verfügen würden, um auf diese Komponente zuzugreifen. <strong>Bitte kontaktieren Sie Ihren Cluster-Administrator</strong>, um Zugriff anzufordern.",
    "forbiddenPageLinkLabel": "Erfahren Sie mehr über Rollen und Berechtigungen",
```